### PR TITLE
[sailfish-webview] Advise PkgConfig usage properly. Fixes JB#55427

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: doc/index.qdoc [hunk 1]
```diff
@@ -17,11 +18,17 @@ Mozilla gecko.
...
+the spec file you should add line:
+
+\code
+BuildRequires:  pkgconfig(qt5embedwidget)
+\endcode
+
+And in your pro file you should add line:
 
 \code
-CONFIG += link_pkgconfig
```
Comment (rainemak) [inline doc/index.qdoc:24]: As you mentioned, this is needed if you do not sailfishapp. Good to promote usage of that as well.

Comment (rainemak) [reply doc/index.qdoc:24]: At the same having a CONFIG twice doesn't really harm.

Comment (vigejolla) [reply doc/index.qdoc:24]: It does harm - it doesn't work. You can't have both sailfishapp and link_pkgconfig there.

Comment (vigejolla) [reply doc/index.qdoc:24]: But I'll rewrite this section so that it mentions both

Comment (rainemak) [reply doc/index.qdoc:24]: Thank @vigejolla :+1:

**Review by rainemak — APPROVED** [review]

**Pull request merged**